cumulus/parachains/runtimes/bridge-hubs/common/src/barriers.rs

```diff
@@ -55,3 +57,35 @@ where
 		Ok(())
 	}
 }
+
+/// Allow unpaid execution from location L when exporting to network N
+pub struct AllowExplicitUnpaidExecutionFromAssetHubExportToEthereum<L, N>(PhantomData<(L, N)>);
```

Comment:
since source and dest are generic params, you could use a generic name

```suggestion
pub struct AllowExplicitUnpaidExecutionForExport<L, N>(PhantomData<(L, N)>);
```

Reply:
actually this is better: https://github.com/paritytech/polkadot-sdk/pull/8599#discussion_r2104380961

Reply:
https://github.com/paritytech/polkadot-sdk/pull/8599/commits/d0cadac0235d2fa586323476b8ce966104c0ab3b